### PR TITLE
Fixes 24479 - fix csv tasks export

### PR DESCRIPTION
**Pull request by iNecas**

### CODE REVIEW

File: app/models/foreman_tasks/task.rb [hunk 1]
```diff
@@ -85,8 +85,26 @@ def execution_type
       delayed? ? N_('Delayed') : N_('Immediate')
     end
 
+    def get_humanized(method)
```
Comment (iNecas) [inline app/models/foreman_tasks/task.rb]: Needed to define this for tests to work properly: we've already seen this issue in Katello, so I though it's better to fix it right way.

File: test/controllers/tasks_controller_test.rb [hunk 2]
```diff
@@ -24,6 +24,14 @@ def in_taxonomy_scope(organization, location = nil)
         Organization.current = Location.current = nil
       end
 
+      it 'supports csv export' do
```
Comment (iNecas) [inline test/controllers/tasks_controller_test.rb]: This should have been done from the beginning of the feature: we would have found the issue much faster than without it.

File: app/models/foreman_tasks/task.rb [hunk 3]
```diff
@@ -85,8 +85,26 @@ def execution_type
...
+      attr = case method
+             when :humanized_name
+               :action
+             when :humanized_input
+               :input
+             when :humanized_output
+               :output
+             else
+               nil
```
Comment (adamruzicka) [inline app/models/foreman_tasks/task.rb]: `case` will implicitly return `nil` if none of the branches match. That's what rubocop complains about

**Comment by iNecas** [comment]
Updated

**Comment by adamruzicka** [comment]
Test failure seems unrelated, ACK

**Comment by adamruzicka** [comment]
Thanks @iNecas !

**Pull request merged**